### PR TITLE
Clarify instructions for configuring Danger on projects

### PR DESCRIPTION
**Pull request by digitalronin**
Clarify the instructions for adding Danger to MoJ projects. This includes
feedback from comments in #6

### CODE REVIEW

**Comment by digitalronin** [comment]
This is branched off "convert-to-gem" so the addition of the gemspec file will not be a part of this commit, by the time it's merged (assuming #7 is merged before this PR).

File: README.md [hunk 1]
```diff
@@ -4,33 +4,38 @@
...
 
-## Setup
+[![Gem Version](https://badge.fury.io/rb/ministryofjustice-danger.svg)](https://badge.fury.io/rb/ministryofjustice-danger)
 
-Enable Danger for a project within the [ministryofjustice
-organization](https://github.com/ministryofjustice).
+# Setup
 
-### Configure Danger to use this repo's Dangerfile
+## Add Danger to your [ministryofjustice organization](https://github.com/ministryofjustice) Ruby project
```
Comment (SteveMarshall) [inline README.md]: Probably don't need the word `organisation` in here?

**Review by SteveMarshall — DISMISSED** [review]
:shipit: (one minor wording tweak maybe?)

**Comment by SteveMarshall** [comment]
Also, the title of this PR (and the commit message) say *what* you've done, not why. A better commit message and PR title might be “Clarify instructions for configuring Danger on projects”?

**Review by SteveMarshall — DISMISSED** [review]
:shipit:

**Review by SteveMarshall — DISMISSED** [review]
LGTM! Rebate against master and merge?

**Review by SteveMarshall — APPROVED** [review]
😎

**Pull request merged**